packages/shared/chat/sagas.ts

```diff
@@ -71,9 +72,12 @@ function* trackEvents(action: PayloadAction<MessageEvent, ChatMessage>) {
   const { type, payload } = action
   switch (type) {
     case SEND_MESSAGE: {
-      trackEvent('Control Send chat message', {
+      trackEvent('Send chat message', {
         messageId: payload.messageId,
+        from: hasWallet() ? ChatMessagePlayerType.WALLET : ChatMessagePlayerType.GUEST,
+        to: ChatMessagePlayerType.WALLET,
```

Comment:
Remember this could also be a public chat (currently `#nearby`) sent through comms, please check that the `payload.messageType === Private` for this to be WALLET, else it might be better to set this to `undefined` or another enum value like `Nearby`.

Reply:
Done!